core/src/test/java/com/radixdlt/recovery/REv2ConsensusLedgerRecoveryTest.java

```diff
@@ -131,8 +130,8 @@ public void recovery_should_work_when_consensus_is_behind_ledger() {
       test.runNext(
           msg -> msg.message() instanceof Proposal && msg.channelId().isLocal(behindNodeIndex));
 
-      // Assert: Can run bft syncing with no issue
-      test.runForCount(100, onlyBFTSyncEvents());
+      // Assert: Can run consensus with no issue
+      test.runUntilState(allAtOrOverStateVersion(50), 300, onlyConsensusEvents());
```

Comment:
We don't prune the vertex store state anymore, so there's no need to sync after reboot. Instead, making sure that consensus runs fine.